build/js/live-editor.debugger.js

```diff
@@ -590,34 +634,8 @@ window.ScratchpadDebugger = Backbone.View.extend({
         this.editor = options.editor;
         this.liveEditor = options.liveEditor;
 
-        if (this.isSupported()) {
-            this.render();
-            this.bind();
-        }
-    },
-
-    isSupported: function() {
-        // Source:
-        // https://github.com/kangax/compat-table/blob/gh-pages/data-es6.js#L1554-L1564
-        try {
-            var code = "\n" +
-                "var generator = (function* () {\n" +
-                "  yield* (function* () {\n" +
-                "    yield 5; yield 6;\n" +
-                "  }());\n" +
-                "}());\n" +
-                "\n" +
-                "var item = generator.next();\n" +
-                "var passed = item.value === 5 && item.done === false;\n" +
-                "item = generator.next();\n" +
-                "passed &= item.value === 6 && item.done === false;\n" +
-                "item = generator.next();\n" +
-                "passed &= item.value === undefined && item.done === true;\n" +
-                "return passed;";
-            return Function(code)();
-        } catch (e) {
-            return false;
-        }
```

Comment:
we don't need to check if generators are supported anymore because we're using regenerator now  ^_^